### ISSUE TITLE
Add LetsEncrypt support

### ISSUE THREAD

**Issue by rasteiner** [rasteiner/k3-dockercompose-starterkit#3]

**Comment by joernroeder** [comment]
i'm having a similar setup, running kirby panel container alongside nuxt container with an nginx-reverse proxy in front of it and added letsencrypt support via https://github.com/jwilder/nginx-proxy
it has the great [lets encrypt companion](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion). was easy to setup and works well!

**Comment by rasteiner** [comment]
Thx for the pointer :)

As far as you know, is there much difference between `nginx` and `nginx proxy`? I mean could I use the "LetsEncrypt companion" also for my `ngnix not-proxy`? 

Otherwise I thought about following this tutorial: https://medium.com/@pentacent/nginx-and-lets-encrypt-with-docker-in-less-than-5-minutes-b4b8a60d3a71
(I'm doubtful about the "less than 5 minutes" part, but I think it should work).

**Comment by joernroeder** [comment]
@rasteiner yes, accoring to the docs you can also run it alongside the official `nginx` container -> https://github.com/jwilder/nginx-proxy#separate-containers

I really like the idea of having a single entrypoint (container) which binds to the public network interface and handles TLS, gzip, CORS-Headers, potential load balancing, multi domains etc and just forwards to the corresponding container. then i dont have to take care about that public network stuff in individual backends.
